utils/configMapper.py

```diff
@@ -9,6 +9,7 @@
 
 log = logging.getLogger("configMapper")
 
+
```

Comment:
global: format only changes please revert in untouched files. if we want to update style do it on purpose I'm another Mr for style clean up. applys to other files in review also.

Reply:
this file may have other changes. If so we should talk about differing styles and auto formatters.